ecommerce-ui/src/Rental.js

```diff
@@ -0,0 +1,118 @@
+import React, { Component } from 'react';
+import {CardColumns,Card, CardDeck, CardGroup, CardImg, ListGroup, ListGroupItem, Button,Jumbotron} from 'react-bootstrap';
+import ReactStars from 'react-stars';
+import PropTypes from 'prop-types';
```

Comment:
You'll want to continue using proptypes, at the very least for the rental component.